docs/EN_US/ECLWatch/TheECLWatchMan.xml

```diff
@@ -29,8 +37,9 @@
       <para>LexisNexis and the Knowledge Burst logo are registered trademarks
       of Reed Elsevier Properties Inc., used under license.</para>
 
-      <para><!-- DNT-Start -->HPCC<!-- DNT-End --> Systems<superscript>®</superscript> is a registered trademark
-      of LexisNexis Risk Data Management Inc.</para>
+      <para><!-- DNT-Start -->HPCC<!-- DNT-End -->
+      Systems<superscript>®</superscript> is a registered trademark of
+      LexisNexis Risk Data Management Inc.</para>
```

Comment:
I am not familiar with "LexisNexis Risk Data Management Inc".   Are we sure about this?

Reply:
LexisNexis Risk Data Management Inc is correct. I just reconfirmed from the internal trademark list to be sure.